m3.py

```diff
@@ -4,18 +4,21 @@
 import click
 
 from src.cli.add import Add
+from src.cli.apply import Apply
 from src.cli.auth import Auth
 from src.cli.init import Init
 from src.cli.list import List
 from src.config.config import Config
 
+CONTEXT_SETTINGS = {'help_option_names': ['-h', '--help']}
```

Comment:
This is short enough to be inline.

Only do this for long multiline args like helptext.